### ISSUE TITLE
Presets should have “warning” instead of “error” severity

### ISSUE THREAD

**Issue by lensbart** [azat-io/eslint-plugin-perfectionist#221]
First of all, thanks for maintaining this project, I appreciate it very much!

### What rule do you want to change?

All three presets

### Describe the problem

ESLint severity levels can serve a purpose, and when all ESLint problems are reported as errors, some signal gets lost. A useful distinction between warnings and errors would be that

- Warnings report stylistic problems that can be resolved with autofix-on-save
- Errors report logic problems (e.g. type errors), and perhaps problems that cannot be autofixed

In a project that maintains a distinction between ESLint warnings and errors, it is distracting to see e.g. `Expected "a" to come before "b" eslint [perfectionist/sort-objects]` being reported as an error.

### Code example

```ts
const { b, a } = something
```

### Additional comments

It is still possible to let CI fail on warnings with `--max-warnings 0`

### Validations

- [X] Read the [docs](https://perfectionist.dev).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Hello, thank you for your issue!

We do not plan to change severity in our configs.

If you need to use warning instead of error, you can do it manually:

```js
import perfectionist from 'eslint-plugin-perfectionist'

export default [
  {
    plugins: {
      perfectionist,
    },
    rules: {
      'perfectionist/sort-jsx-props': ['warning'],
      'perfectionist/sort-objects': ['warning'],
    },
  },
]
```
